models/models.go

```diff
@@ -1,6 +1,9 @@
 package models
 
 type User struct {
+	Id  int	`json:"id,gt=0"`
```

Comment:
ID not required here, it is automatically set by Firestore when we insert a user.